TestAssemblies/AssemblyWithPropertyAttributes/ClassWithAttributedProperties.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Collections.Generic;
+using System.ComponentModel;
+using PropertyChanged;
+
+public class ClassWithAttributedProperties : INotifyPropertyChanged
```

Comment:
comments here and all classes
we'll need to support this code forever so good to have it fully commented up and keep wiki updated